README.md

```diff
@@ -9,37 +9,49 @@ A polyglot messaging service.
 [![License](https://img.shields.io/npm/l/sockethub.svg?style=flat)](https://raw.githubusercontent.com/sockethub/sockethub/master/LICENSE)
 [![Release](https://img.shields.io/npm/v/sockethub.svg?style=flat)](https://github.com/sockethub/sockethub/releases)
 
-[![Flattr this](http://api.flattr.com/button/flattr-badge-large.png)](http://flattr.com/thing/1154379/Sockethub)
-
 # About
-Sockethub is a polyglot (speaking many different protocols and APIs) messaging service for social and other interactive messaging applications. Not only does it assist [unhosted](http://unhosted.org) and [noBackend](http://nobackend.org) web application developers by providing server-independent, server-side functionality, but it also can be used as a back-end tool (invisible to the user) for many different applications, large and small.
+Sockethub is a translation layer for web-apps to communicate with other protocols and services that are traditionally inaccessible or impracticle.
+
+Using [ActivityStream](http://activitystrea.ms/) (AS) objects to pass messages to and from the web-app, Sockethub acts as a sort of smart web-proxy and can maintain state, connect to sockets or REST enpoints that would otherwise be restricted from an application running in the browser. Originally inspired as a sister project to [RemoteStorage](https://remotestorage.io) and assisting in the development of [unhosted](http://unhosted.org) or [noBackend](http://nobackend.org) applications, Sockethubs functionality can also fit into a more traditional development stack, removing the need for custom code to handle various protocol-specifics at the applicaton layer.
```

Comment:
```suggestion
Using [ActivityStream](http://activitystrea.ms/) (AS) objects to pass messages to and from the web app, Sockethub acts as a smart proxy server/agent, which can maintain state, and connect to sockets, endpoints and networks that would otherwise be restricted from an application running in the browser.

Originally inspired as a sister project to [RemoteStorage](https://remotestorage.io), and assisting in the development of [unhosted](http://unhosted.org) and [noBackend](http://nobackend.org) applications, Sockethub's functionality can also fit into a more traditional development stack, removing the need for custom code to handle various protocol specifics at the application layer.
```